cmd/av/stack_sync.go

```diff
@@ -141,23 +143,26 @@ base branch.
 			return nil
 		}
 
-		// Make sure all changes are staged
-		diff, err := repo.Diff(&git.DiffOpts{Quiet: true})
-		if err != nil {
-			return err
-		}
-		if !diff.Empty {
-			return errors.New("refusing to sync: there are unstaged changes in the working tree (use `git add` to stage changes)")
+		if !stackSyncFlags.Skip {
+			// Make sure all changes are staged unless --skip. git rebase --skip will
+			// clean up the changes.
+			diff, err := repo.Diff(&git.DiffOpts{Quiet: true})
+			if err != nil {
+				return err
+			}
+			if !diff.Empty {
+				return errors.New("refusing to sync: there are unstaged changes in the working tree (use `git add` to stage changes)")
+			}
 		}
 
-		if stackSyncFlags.Continue {
+		if stackSyncFlags.Continue || stackSyncFlags.Skip {
 			if state.CurrentBranch == "" {
 				return errors.New("no sync in progress")
 			}
 		} else {
-			// Not a --continue, we're trying to start a new sync from scratch.
+			// Not a --continue/--skip, we're trying to start a new sync from scratch.
 			if state.CurrentBranch != "" {
-				return errors.New("a sync is already in progress: use --continue or --abort")
+				return errors.New("a sync is already in progress: use --continue, --abort, or --skip")
```

Comment:
I'd actually rather not suggest `--skip` to the average user. Generally people aren't going to want to do this very often and I think presenting it as an option at the same level as `--continue` or `--abort` will lead people to do it when they shouldn't.

Reply:
Sure. This is for addressing cases where merge conflict happens for the already merged branches. I usually use --skip because they are already merged, but I can remove --here. (Not sure what people should do for those cases otherwise.)